src/network/networkprotocol.h

```diff
@@ -942,16 +942,16 @@ enum NetProtoCompressionMode {
 };
 
 const static std::string accessDeniedStrings[SERVER_ACCESSDENIED_MAX] = {
-	"Invalid password",
-	"Your client sent something the server didn't expect.  Try reconnecting or updating your client",
-	"The server is running in simple singleplayer mode.  You cannot connect.",
-	"Your client's version is not supported.\nPlease contact server administrator.",
-	"Player name contains disallowed characters.",
-	"Player name not allowed.",
-	"Too many users.",
-	"Empty passwords are disallowed.  Set a password and try again.",
-	"Another client is connected with this name.  If your client closed unexpectedly, try again in a minute.",
-	"Server authention failed.  This is likely a server error."
+	"You have entered a wrong password.",
+	"Your client sent something the server didn't expect. Try reconnecting or updating your client.",
+	"The server is running in simple singleplayer mode. You cannot connect.",
+	"Your client's version is not supported.\nPlease contact a server administrator.",
+	"The player name contains unallowed characters. Allowed characters are a-z, A-Z, 0-9, hyphen and underscore.",
```

Comment:
unallowed isn't a word: http://dictionary.cambridge.org/spellcheck/american-english/?q=unallowed
disallowed is correct.